HW1/P2/P2a.py

```diff
@@ -1,3 +1,71 @@
-from P2 import *
+# shut down the previous spark context
+#sc.stop() 
+sc = pyspark.SparkContext(appName="myAppName")
+import numpy as np
+import matplotlib.pyplot as plt 
+import matplotlib.cm as cm
+import pdb
 
-# Your code here
+def mandelbrot(x, y):
```

Comment:
You don't need to copy all these helper functions into P2a.py, instead you can just import them using from P2 import *